braindecode/datasets/sleep_physionet.py

```diff
@@ -23,13 +23,14 @@ class SleepPhysionet(BaseConcatDataset):
 
     Parameters
     ----------
-    subject_ids: list(int) | int
-        (list of) int of subject(s) to be loaded.
-    recording_ids: list(int)
+    subject_ids: list(int) | int | None
+        (list of) int of subject(s) to be loaded. If None, load all available
+        subjects.
+    recording_ids: list(int) | None
         Recordings to load per subject (each subject except 13 has two
-        recordings). Can be [1], [2] or [1, 2].
+        recordings). Can be [1], [2] or [1, 2] (same as None).
     preload: bool
-        if True, preload the data of the Raw objects.
+        If True, preload the data of the Raw objects.
     load_eeg_only: bool
```

Comment:
Is seems that this is currently not used. Do you think it is necessary to have this option? It looks like a shortcut to avoid using ```mne.Raw.drop_channels``` or ```mne.Raw.pick``` / ```mne.Raw.pick_channels``` / ```mne.Raw.pick_types``` in a preprocessing step.

Reply:
Ok, nevermind. It is used and channels are already exluded on read from HDD.